### DISCUSSION TITLE
How do I download the files at /var/lib/pterodactyl/volumes?

### DISCUSSION THREAD

**Discussion by DumbCaveSpider** [pterodactyl/panel#3446] [Support]
Is there anyway to give myself permission to access `/var/lib/pterodactyl/volumes`. I need to download the files through `FileZilla`. If there's a way, what's the command for this. Thanks!

![image](https://user-images.githubusercontent.com/56347227/123630330-fa62f900-d847-11eb-9f36-f5f7641c420f.png)

**Comment by TekExplorer** [comment]
you would need to be logged in as root, or mess with directory/user permissions.

**Comment by Tetroner9** [comment]
still got the solution?

**Comment by OddDevelopment** [comment]
You need permissions so I would recommend using root or an account with admin permissions when logging in using SFTP and that should fix this issue.

Also for a quicker download you should compress the volumes folder using a command like `nohup tar -czf backup.tar.gz ./volumes > /dev/null 2>&1 &` this will save time and bandwidth.
